dexbot/strategies/external_feeds/ccxt_feed.py

```diff
@@ -143,28 +125,14 @@ def test_feed(exchange, symbol):
 
 @main.command()
 def test_exch_list():
-    '''
+    """
     gets a list of supported exchanges
-    '''
+    """
```

Comment:
- Could be just `test_exchange_list`, it would be more clear.
- Same what I mentioned earlier about doc string.